config/default.yaml

```diff
@@ -0,0 +1,22 @@
+#Set ROS publishing rate - independet from the actual frame rate of the camera (it adjust dynamically)
+frame_rate: 20.0
+
+#Activate Downsampling and set the size of the generated voxels in meter. (0.05 = 5cm)
+#This is helpful to reduce the bandwith in the network or to limit the data comming to your algorithm afterwards
+downsampling: false
+downsampling_voxel_size: 0.05
+
+#Enable or Disable the projected range or/and intensity image
+pointcloud: true
+range_image: true
+intensity_image: true
+
+#Settings for the Auto Exposure
+ampl_single: 300.0
+max_exposure: 32766 #multiplied by 0.00001241 is time in seconds
+hdr_factor : 0.5
+ampl_hdr: 800.0
+hdr_mode: false
+
+#Pointcloud scaling
+XYZ_to_m: 0.00037
```

Comment:
The [default value that set here](https://github.com/pieye/nimbus-ros/blob/b386a47aa7f000973885a6d5f91c4373b8ed2906/src/nimbus_ros.cpp#L41) will be changed with this value. According to my calculation, I got 0.00017. Which value we should consider it as an accurate one?

Reply:
Hi Vishni, yeah that can be true. There is a bug in the embedded driver. When you restart the ROS driver after closing it, the factor changes from 0.00037 to 0.00017. Did you experience the same behavior?

Reply:
Yes. Because when measure point cloud distance to a flat surface some time show perfect distance other time it completely faulty measurment. What we have to do to avoid this?

Reply:
We're working on fixing this.
The issue only occurs when you restart your raspberry and start the ROS driver the **first** time.
So a workaround, for now, would be to use your factor and make sure to not use the data when you started the ROS driver the first time until boot up.

Reply:
Just to keep you updated: It is a bug in the Raspian Kernel, Markus is working on a fix.